onyo/lib/items.py

```diff
@@ -319,27 +319,34 @@ def get_path_name(self) -> Path | None:
 
     def is_asset(self) -> bool | None:
         """Initializer for the ``'onyo.is.asset'`` pseudo-key."""
-
         if not self.repo or not self._path:
             return None
 
-        return self.repo.is_asset_path(self._path)
+        # True, if it's either an existing asset in the inventory or
+        # it's "instructions" for creating one. The latter implies there
```

Comment:
> it's "instructions" for creating one. The latter implies there
> are non-pseudokeys (or "onyo.is.asset" is defined in it and would overwrite
> this evaluation anyway)

it has "instructions" to create one (e.g. a template or YAML file containing "onyo.is.asset: True").

Reply:
Chaged the comment to hopefully make it a bit clearer.